### ISSUE TITLE
Notes should be sorted by order of creation if on the same day

### ISSUE THREAD

**Issue by sleidig** [Aam-Digital/ndb-core#2509]
**Describe the bug**
If you create multiple notes for the same day, the most recently created record is not showing on top of the list (we only sort by date, without any time information). This is very confusion to users.

**To Reproduce**

1. Go to "Notes"
2. Add a new note for the same date of the last note in the list
3. Find your new note (it's probably not at the top of the list where you would expect it)

**Expected behavior**
Ideally the notes of the same day should be sorted "sensibly". Can we use the "created.at" timestamp as a secondary sorting criteria for those that have the same date, maybe?

**Comment by sleidig** [comment]
Yes, underlying all the lists is `EntitiesTableComponent`. I think we have some special logic to transform columns into sortable values already in src/app/core/common-components/entities-table/table-sort/table-sort.ts, maybe that is a good place to look into

**Comment by Abhinegi2** [comment]
@george-neha could you please test the above-mentioned issue here-  https://pr-2925.aam-digital.net/?
